recipe/build.sh

```diff
@@ -1,4 +1,12 @@
-cmake -DBUILD_THIRDPARTY=ON -DCMAKE_INSTALL_PREFIX=$PREFIX .
+#!/bin/bash
+
+if [ $(uname) == Darwin ]; then
+    SONAME="dylib"
+else
+    SONAME="so"
+fi
+
+cmake -D CMAKE_INSTALL_PREFIX=$PREFIX -D TIFF_LIBRARY=$PREFIX/lib/libtiff.${SONAME} -D TIFF_INCLUDE_DIR=$PREFIX/include -D PNG_LIBRARY_RELEASE=$PREFIX/lib/libpng.${SONAME} -D PNG_PNG_INCLUDE_DIR=$PREFIX/include -D ZLIB_LIBRARY=$PREFIX/lib/libz.${SONAME} -D ZLIB_INCLUDE_DIR=$PREFIX/include .
```

Comment:
Sorry, I wasn't very clear. I should have said I like the way you have done line wraps here, but it would be nice if the options line up. Like so...

    cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
          -D TIFF_LIBRARY=$PREFIX/lib/libtiff.${SONAME} \